docs/rules/order-in-routes.md

```diff
@@ -84,8 +84,7 @@ You should write code grouped and ordered in this way:
 ```javascript
 const {
   Route,
-  inject: { service },
-  get
+  inject: { service }
 } = Ember;
```

Comment:
Should probably change these to imports at some point also (possibly in a future PR though).

Reply:
Good point, I'll follow-up with that in a separate PR.

Reply:
Good point, addressed in this PR: https://github.com/ember-cli/eslint-plugin-ember/pull/748